packages/ui-patterns/ConsentToast/index.tsx

```diff
@@ -25,15 +26,17 @@ export const ConsentToast = ({ onAccept = noop, onOptOut = noop }: ConsentToastP
     <div className="py-1 flex flex-col gap-y-3 w-full">
       <div>
         <p className="text-sm text-foreground">
-          We use first-party cookies to improve our services.
+          We use{' '}
           <Link
-            className="inline sm:hidden underline text-light"
-            target="_blank"
-            rel="noreferrer"
-            href="https://supabase.com/privacy"
+            href="https://supabase.com/privacy#8-cookies-and-similar-technologies-used-on-our-european-services"
+            className="inline hover:underline"
```

Comment:
now sure how i should handle this without looking like the banner has too many underlined items?